### ISSUE TITLE
Tag newer versions

### ISSUE THREAD

**Issue by leonty** [jboss-dockerfiles/jbpm#58]
Please add 7.19.0 and 7.20.0 releases.

**Comment by kenjiuno** [comment]
Yes, I also want to see 7.20.0 release in Docker: `docker pull jboss/jbpm-server-full`
https://hub.docker.com/r/jboss/jbpm-server-full/dockerfile still points `jBPM Workbench 7.18.0.Final`, `KIE Server 7.18.0.Final` and so on...

**Comment by sazonov** [comment]
Hi! 7.21.0.Final has been released. @mbiarnes Could you please update the docker image?

**Comment by sazonov** [comment]
@mbiarnes Hi! Could you please tell if the update is planned? We have a blocker issue with the process designer on 7.18.0.Final. Thanks.

**Comment by hmatt1** [comment]
One of the beauties of open source is that you can build the project yourself if you need a specific version. For example, I wanted to run with the current latest version, `7.31.0.Final`.

You can check out this commit to see where you will need to change the versions: https://github.com/jboss-dockerfiles/jbpm/commit/607d243c3d7158bd0454c508e7069432e09fd015

There are various `build.sh` scripts you can run to build the image you need. Just edit the tag! https://github.com/jboss-dockerfiles/jbpm/blob/master/base/build.sh#L8

You could also retag the built image and push it to your company's internal docker registry, something like:
```
docker tag jboss/jbpm-workbench-showcase:7.31.0.Final docker.mycompany.com/my-project/jbpm-workbench-showcase:7.31.0.Final
docker login docker.mycompany.com
docker push docker.mycompany.com/my-project/jbpm-workbench-showcase:7.31.0.Final
```
